### DISCUSSION TITLE
Setup for MKDocs

### DISCUSSION THREAD

**Discussion by futuresoup** [decaporg/decap-cms#6768] [Q&A]
Hi, I'm a non-developer using MKDocs for my [open source project](https://github.com/climate-tech-handbook/climate-tech-handbook).

I noticed help guides on the website for other SSGs, but not MKDocs.

Any guidance would be wonderful! Thanks!

**Comment by martinjagodic** [comment]
I'm not familiar with MKDocs, but Decap is SSG-agnostic, so it should work with it like any other SSG. The instructions are here: https://decapcms.org/docs/add-to-your-site/

**Comment by alexmayberry** [comment]
We implemented MKDocs w/ Decap. Check it out! https://github.com/climate-tech-handbook/climate-tech-handbook
